lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -609,7 +609,11 @@ defmodule ExUnit.Assertions do
       assert_in_delta 10, 15, 4
 
   """
-  def assert_in_delta(value1, value2, delta, message \\ nil) do
+  def assert_in_delta(value1, value2, delta, message \\ nil)
+  def assert_in_delta(_, _, delta, _) when delta < 0 do
+    raise ArgumentError, "all deltas must be positive. Did you mean #{abs(delta)}?"
```

Comment:
I don't think this is a good error message with the "did you mean" thing. I think we should have the user decide however they want to fix this error, for example they might want to use `abs/1` even if this is negative and we know it because they might want both positive and negative numbers to work. I believe the correct error message here is what we almost always say, which is something along the lines of

```
delta must always be a positive number, got: #{inspect(delta)}
```

Wdyt?